espressif/vscode-esp-idf-extension
# v1.8.0 release

## Description

Add changes for release v1.8.0

Fix #1227 
Fix #1208 
Fix #1210
Fix #1213 
Fix #1217

## Type of change

- This change requires a documentation update
- Bug fixes 

## Steps to test this pull request

1. Test `ESP-IDF: Install ESP-Matter` without any folder opened. Matter should be installed without issues and `idf.espMatterPath` should be set and examples shown.
2. Create a new ESP-IDF project and launch `Start OpenOCD`, the settings from workspace folder should be used without updating settings.json.
A Complete test of the extension is required.

## How has this been tested?


**Test Configuration**:
* ESP-IDF Version: 5.2
* OS (Windows,Linux and macOS): macOS

## Checklist
